src/NUnitFramework/framework/Internal/Execution/WorkItemQueue.cs

```diff
@@ -101,17 +101,28 @@ public SavedState(WorkItemQueue queue)
         private int _addId = int.MinValue;
         private int _removeId = int.MinValue;
 
+#if COM_APARTMENT
```

Comment:
This could be moved to line 109/110 to prevent the duplicate documentation.

Reply:
The weird thing is that you can't put preprocessor inside XML documentation. I thought it was simple but https://github.com/dotnet/csharplang/issues/295#issuecomment-327967932 blew my mind.